src/test/ui/error-codes/E0648.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0648]: `export_name` may not contain null characters
+  --> $DIR/E0648.rs:11:1
+   |
+LL | #[export_name="/0foo"] //~ ERROR E0648
```

Comment:
This output looks like the slash was inverted - is that an artifact of compiletest?

Reply:
Yeah — the actual error is correct, but for whatever reason, the stderr requires a forward slash.

Reply:
It's probably because we're trying to have all paths be unix-style and are perhaps a bit too eager with slash replacement.